mainnet-1/specs/evmos.json

```diff
@@ -0,0 +1,1135 @@
+                                    "stateful": 0
+                                },
+                                "extra_compute_units": 0
+                            }
+                        ],
+                        "headers": [],
+                        "inheritance_apis": [],
+                        "parse_directives": [],
+                        "verifications": [
+                            {
+                                "name": "minimum-gas-price",
+                                "values": [
+                                    {
+                                        "expected_value": "50000000000.000000000000000000aevmos",
+                                        "severity": "Warning"
+                                    }
+                                ]
+                            },
+                            {
+                                "name": "chain-id",
+                                "values": [
+                                    {
+                                        "expected_value": "evmos_9001-2"
+                                    }
+                                ]
+                            },
+                            {
+                                "name": "pruning",
+                                "values": [
+                                    {
+                                        "latest_distance": 22154
+                                    },
+                                    {
+                                        "extension": "archive",
+                                        "latest_distance": 1000000
+                                    }
+                                ]
+                            }
+                        ],
+                        "extensions": [
+                            {
+                                "name": "archive",
+                                "cu_multiplier": 5,
+                                "rule": {
+                                    "block": 21923
+                                }
+                            }
+                        ]
+                    },
+                    {
+                        "enabled": true,
+                        "collection_data": {
+                            "api_interface": "tendermintrpc",
+                            "internal_path": "",
+                            "type": "",
+                            "add_on": ""
+                        },
+                        "apis": [],
+                        "headers": [],
+                        "inheritance_apis": [],
+                        "parse_directives": [],
+                        "verifications": [
+                            {
+                                "name": "minimum-gas-price",
+                                "values": [
+                                    {
+                                        "expected_value": "CiQ1MDAwMDAwMDAwMC4wMDAwMDAwMDAwMDAwMDAwMDBhZXZtb3M",
```

Comment:
This value should be udpated to: "CiQ1MDAwMDAwMDAwMC4wMDAwMDAwMDAwMDAwMDAwMDBhZXZtb3M="

is missing the "=" sign at the end.